types/gsi/index.d.ts

```diff
@@ -0,0 +1,271 @@
+// Type definitions for non-npm package gsi (Sign In With Google JavaScript API) 0.0
```

Comment:
```suggestion
// Type definitions for non-npm package Sign In With Google JavaScript API
```

Reply:
removing version number will break the `dt-header` ts-lint rule

Reply:
added back to suggestion, you're correct